frontend/src/metabase/admin/routes.jsx

```diff
@@ -121,6 +122,7 @@ const getRoutes = (store, IsAdmin) => (
         />
       </Route>
       <Route path="logs" component={Logs} />
+      <Route path="bug-report" component={BugReportApp} />
```

Comment:
Super minor, but most of our URLs don't have spaces in them, and the ones that do use an underscore. We should probably be consistent for consistency's sake.

Reply:
Let's go ahead and change this from `-` to `_`